claragenomics/dl4atac/dataset.py

```diff
@@ -8,64 +8,92 @@
 # license agreement from NVIDIA CORPORATION is strictly prohibited.
 #
 
-import numpy as np
+"""Dataset class. Load data for training, inference, evaluation."""
```

Comment:
@avantikalal  please review the documentation of all modules and functions in this file. I'm not confident about it.